SimpleChat.Client/wwwroot/ts/pagination.ts

```diff
@@ -0,0 +1,11 @@
+﻿
+function addPaginationEvent(event) {
```

Comment:
Try to avoid using "any" type. Give a type to your parameter. And change it name

Reply:
What type will look right?